package.json

```diff
@@ -42,7 +42,8 @@
   },
   "scripts": {
     "bench": "matcha",
-    "test": "standard && tape test/*.test.js | tap-spec"
+    "test": "standard && tape test/*.test.js | tap-spec",
+    "test:new": "tape null-values.test.js | tap-spec"
```

Comment:
Oh boy, I forgot to remove that again – just wanted to use it to run that particular one in quick succession without the noisy output from the other tests, but then I never did